### ISSUE TITLE
Tune hyperparameters for best results

### ISSUE THREAD

**Issue by ERUD1T3** [ERUD1T3/artificial-neural-network#6]
- Tune learning rate
- Tune momentum
- Tune decay
- Tune number of epochs 
- Tune number of hidden layers ...

to achieve best performance on the following data set

- [x] Identity
- [x] Iris 
- [x] Tennis
